XenAdmin/Wizards/PatchingWizard/PatchingWizard_PrecheckPage.cs

```diff
@@ -400,6 +400,8 @@ private void worker_DoWork(object sender, DoWorkEventArgs e)
                         ? Updates.GetUpgradeSequence(pool.Connection, UpdateAlert, ApplyUpdatesToNewVersion && !automatedUpdatesRestricted)
                         : Updates.GetUpgradeSequence(pool.Connection);
 
+                    log.InfoFormat("Minimal patches for {0}: {1}", pool.Name, string.Join(",", us.MinimalPatches.Select(p => p.Name)));

```

Comment:
I think this should definitely be after the null check on 'us' (in next line). Additionally, a null check for the MinimalPatches is also needed here.

Reply:
...or we can just leave it here, but do the null-check inline (to log that case too)

Reply:
Fixed in the new commit